core/server/web/api/canary/admin/routes.js

```diff
@@ -75,6 +75,7 @@ module.exports = function apiRoutes() {
 
     router.put('/users/password', mw.authAdminApi, http(apiCanary.users.changePassword));
     router.put('/users/owner', mw.authAdminApi, http(apiCanary.users.transferOwnership));
+    router.put('/users/reset_all_passwords', mw.authAdminApi, http(apiCanary.users.resetAllPasswords));
```

Comment:
@tpatel imo this would make more sense as an auth endpoint like `/authentication/reset` - we already have `passwordreset` there: https://github.com/TryGhost/Ghost/blob/5ef9bdc91a36dbf46a413ce70b2f34e6e6fc495f/core/server/web/api/canary/admin/routes.js#L211

Reply:
Yes I was hesitating between the two with no strong opinion, I'm gonna move it under `/authentication` then 👍